src/shared/utils/upgradeForm.ts

```diff
@@ -236,19 +234,24 @@ export const getDefaultValuesUpgradeForm = ({
     plans,
   })
 
-  const isMonthlyPlan = accountDetails?.plan?.billingRate === 'monthly'
+  const isMonthlyPlan =
+    accountDetails?.plan?.billingRate === BillingRate.MONTHLY
+  const isPaidPlan = !!accountDetails?.plan?.billingRate // If the plan has a billing rate, it's a paid plan
```

Comment:
The only part of this diff that really matters is the new isPaidPlan const which is doing what the helper function was doing previously... this was also the only place that helper was used 😅